src/referencePackages/src/microsoft.codeanalysis.visualbasic/4.0.1/lib/netstandard2.0/Microsoft.CodeAnalysis.VisualBasic.cs

```diff
@@ -47,10 +47,8 @@
 [assembly: System.Reflection.AssemblyFlagsAttribute((System.Reflection.AssemblyNameFlags)0x70)]
 namespace Microsoft.CodeAnalysis
 {
-    public sealed partial class VisualBasicExtensions
+    public static partial class VisualBasicExtensions
     {
-        internal VisualBasicExtensions() { }
-
```

Comment:
Is there a bug logged for the incorrect generation here?

Reply:
https://github.com/dotnet/source-build/issues/3495